src/main/java/org/zalando/nakadi/service/CursorOperationsService.java

```diff
@@ -44,7 +44,7 @@ public Long calculateDistance(final NakadiCursor initialCursor, final NakadiCurs
         if (!initialCursor.getPartition().equals(finalCursor.getPartition())) {
             throw new InvalidCursorOperation(CURSORS_WITH_DIFFERENT_PARTITION);
         } else if (initialCursor.getTimeline().getOrder() > finalCursor.getTimeline().getOrder()) {
-            throw new InvalidCursorOperation(INVERTED_TIMELINE_ORDER);
+            return - getDistanceDifferentTimelines(finalCursor, initialCursor);
```

Comment:
The code I had implemented expects that parameters are correctly sorted. So I just inverted the parameters order and marked the result as negative.